pkg/operator/controller/networkpolicyregistrar/add.go

```diff
@@ -35,9 +37,33 @@ func (r *Reconciler) AddToManager(mgr manager.Manager) error {
 	return builder.
 		ControllerManagedBy(mgr).
 		Named(ControllerName).
-		For(&operatorv1alpha1.Garden{}, builder.WithPredicates(predicateutils.ForEventTypes(predicateutils.Create, predicateutils.Update))).
+		For(&operatorv1alpha1.Garden{}, builder.WithPredicates(NetworkingPredicate())).
 		WithOptions(controller.Options{
 			MaxConcurrentReconciles: 1,
 		}).
 		Complete(r)
 }
+
+// NetworkingPredicate returns true for Create and Update events if the
+// garden.Spec.RuntimeCluster.Networking.{Pods,Services} fields are not empty
```

Comment:
Can you add a TODO to remove this again in `v1.75`? This is only needed for migration/upgrade now, but since the fields are required, they should always be set when new `Garden` resources are created, right?